apps/prairielearn/src/tests/sync/util.ts

```diff
@@ -176,6 +176,7 @@ export interface Question {
   title: string;
   topic: string;
   tags?: string[];
+  authors?: string[];
   clientFiles?: string[];
```

Comment:
why do the tests have entirely separate type declarations for the JSON file schemas than the main codebase has in course-db.ts?

Reply:
or maybe it just changes the permissions on them somehow? anyway, whatever it is doing, git is not happy about it

Reply:
Because they are fundamentally separate things. What you commented on here are types for the JSON file itself; what exists in `course-db.ts` correspond directly to database tables and columns.

Reply:
oh, I see. So this could be seen as a duplicate declaration to the JSON files that define the schemas of the `info.json` files? (in the `schemas` folder)